src/main/java/seedu/address/storage/ShortcutCommandUtil.java

```diff
@@ -0,0 +1,127 @@
+    public static final Path EDIT_PATH = Paths.get("data", "editCommand.txt");
+    public static final Path EXIT_PATH = Paths.get("data", "exitCommand.txt");
+    public static final Path EXPORT_PATH = Paths.get("data", "exportCommand.txt");
+    public static final Path FILTER_PATH = Paths.get("data", "filterCommand.txt");
+    public static final Path FIND_PATH = Paths.get("data", "findCommand.txt");
+    public static final Path HELP_PATH = Paths.get("data", "helpCommand.txt");
+    public static final Path IMPORT_PATH = Paths.get("data", "importCommand.txt");
+    public static final Path LIST_PATH = Paths.get("data", "listCommand.txt");
+    public static final Path MASS_OP_PATH = Paths.get("data", "massOpCommand.txt");
+    public static final Path REDO_PATH = Paths.get("data", "redoCommand.txt");
+    public static final Path SHORTCUT_PATH = Paths.get("data", "shortcutCommand.txt");
+    public static final Path TAG_PATH = Paths.get("data", "tagCommand.txt");
+    public static final Path UNDO_PATH = Paths.get("data", "undoCommand.txt");
+
+    /**
+     * Loads user-defined shortcuts from files.
+     */
+    public static void loadShortcuts() {
+        AddCommand.commandWords = loadWords(ADD_PATH, AddCommand.commandWords);
+        ClearCommand.commandWords = loadWords(CLEAR_PATH, ClearCommand.commandWords);
+        DeleteCommand.commandWords = loadWords(DELETE_PATH, DeleteCommand.commandWords);
+        DeleteTagCommand.commandWords = loadWords(DELETE_TAG_PATH, DeleteTagCommand.commandWords);
+        EditCommand.commandWords = loadWords(EDIT_PATH, EditCommand.commandWords);
+        ExitCommand.commandWords = loadWords(EXIT_PATH, ExitCommand.commandWords);
+        ExportCommand.commandWords = loadWords(EXPORT_PATH, ExportCommand.commandWords);
+        FilterCommand.commandWords = loadWords(FILTER_PATH, FilterCommand.commandWords);
+        FindCommand.commandWords = loadWords(FIND_PATH, FindCommand.commandWords);
+        HelpCommand.commandWords = loadWords(HELP_PATH, HelpCommand.commandWords);
+        ImportCommand.commandWords = loadWords(IMPORT_PATH, ImportCommand.commandWords);
+        ListCommand.commandWords = loadWords(LIST_PATH, ListCommand.commandWords);
+        MassOpCommand.commandWords = loadWords(MASS_OP_PATH, MassOpCommand.commandWords);
+        RedoCommand.commandWords = loadWords(REDO_PATH, RedoCommand.commandWords);
+        ShortcutCommand.commandWords = loadWords(SHORTCUT_PATH, ShortcutCommand.commandWords);
+        TagCommand.commandWords = loadWords(TAG_PATH, TagCommand.commandWords);
+        UndoCommand.commandWords = loadWords(UNDO_PATH, UndoCommand.commandWords);
+    }
+
+    /**
+     * Saves user-defined shortcuts to a file.
+     * @param p File path for the new word to be saved in.
+     * @param commandWords ArrayList to be saved into.
+     */
+    public static void saveWords(Path p, List<String> commandWords) {
+        if (!Files.exists(p)) {
```

Comment:
Just a suggestion, this could use seedu.address.commons.util.createIfMissing() to shorten the code, but it's still good.